spec.bs

```diff
@@ -6521,6 +6547,29 @@ a {{Promise}} |p|, and two sequences of steps, covering the parsing of the value
     1. Decrement |auctionConfig|'s [=auction config/pending promise count=].
 </div>
 
+<div algorithm>
+To <dfn>wait until server response promise resolve</dfn> given an [=auction config=] |auctionConfig|:
+
+  1. Wait until either |auctionConfig|'s [=auction config/server response=] is not a {{Promise}},
+     or |auctionConfig|'s [=auction config/aborted=] is true.
+  1. If |auctionConfig|'s [=auction config/aborted=] is true, return failure.
+  1. If |auctionConfig|'s [=auction config/server response=] is failure, return failure.
+  1. Return.
+</div>
+
+<div algorithm>
+To <dfn>handle a server response promise in configuration</dfn> given an [=auction config=] |auctionConfig|,

```

Comment:
This function is only used in one place, and passing in a set of steps is unclear. Can we inline this into where it is called?

Reply:
done.